.travis.yml

```diff
@@ -78,7 +78,7 @@ deploy:
   provider: releases
   api_key: "$GITHUB_TOKEN"
   file_glob: true # allows '*' in file below
-  file: "build/scripts/*.tar.gz"
+  file: "$TRAVIS_BUILD_DIR/build/scripts/*.tar.gz"
```

Comment:
I think this will not work, I got msg that this should be relative path, not absolute. But you'll see.